package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "sass-embedded",
-  "version": "1.58.0",
-  "protocol-version": "1.1.0",
-  "compiler-version": "1.58.0",
+  "version": "1.58.1",
+  "protocol-version": "1.2.0",
+  "compiler-version": "1.58.1",
```

Comment:
Let's wait until Dart Sass 1.58.1 is released for these.

Reply:
It's failing the test expecting these to be 1.58.1 as dart-sass-embedded from main branch is already on 1.58.1.

Reply:
dart-sass-embedded 1.58.1 release was partially broken as linux-arm64 and linux-arm were failing. So we would have to go 1.58.2 directly as next release.